tests/unit/types/document/test_graph_document.py

```diff
@@ -178,6 +178,13 @@ def test_from_dgl_graph(graph):
     assert (graph.adjacency.col == jina_graph.adjacency.col).all()
     assert (graph.adjacency.col == jina_graph.adjacency.col).all()
 
+    from dgl import DGLGraph
```

Comment:
can this be an individual test with a meaningful test (test_convert_no_edges_graph) or something?